NSSNamespace/User_Authentication
# Payment type view model

Details:

Payment type view model has been added to root level of Models folder and contains a property of type Payment type and a custom constructor that passes in ApplicationDbContext as an argument.

To test:

1. Git fetch
2. Git checkout payment-type-view-model and ensure branch contains criteria specified above.
